eng/scripts/Logging-Functions.ps1

```diff
@@ -64,6 +64,8 @@ function LogGroupStart() {
   }
   elseif (Test-SupportsGitHubLogging) {
     Write-Host "::group::$args"
+  } else { 
+    Write-Host "$args"
```

Comment:
Logging group start also shows up in DevOps, why not print it when running locally?